src/main/java/com/github/krzysiekjodlowski/ox/board/Board.java

```diff
@@ -0,0 +1,104 @@
+package com.github.krzysiekjodlowski.ox.board;
+
+import com.github.krzysiekjodlowski.ox.Event;
+import com.github.krzysiekjodlowski.ox.Move;
+import com.github.krzysiekjodlowski.ox.NumberLowerThanOneException;
+import com.github.krzysiekjodlowski.ox.Subscriber;
+import com.github.krzysiekjodlowski.ox.model.FieldNumber;
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Store fields marked by players.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public class Board implements Subscriber {
+  private final Map<FieldNumber, Symbol> fields = new HashMap<>();
+  private final int boardSideLength;
+  private final int boardCapacity;
+
+  public Board(final int boardSideLength) {
+    this.boardSideLength = boardSideLength;
+    this.boardCapacity = this.boardSideLength * this.boardSideLength;
```

Comment:
You have boardCapacity both in settings and here? That's confusing...

Reply:
Removed from settings:) boardSideLength also.